aptos-move/framework/aptos-framework/sources/aptos_coin.move

```diff
@@ -159,4 +159,9 @@ module aptos_framework::aptos_coin {
         aggregator_factory::initialize_aggregator_factory_for_test(aptos_framework);
         initialize(aptos_framework)
     }
+
+    #[test_only]
+    public fun initialize_for_test_without_aggregator_factory(aptos_framework: &signer): (BurnCapability<AptosCoin>, MintCapability<AptosCoin>) {
+        initialize(aptos_framework)
+    }
```

Comment:
I'm just curious, does how does the one with the aggregator factory not support your testing case?

Reply:
+1

Reply:
one cannot initialize genesis and get access to aptos_coin ... we had an oversight.